### ISSUE TITLE
icons blur + transparent line

### ISSUE THREAD

**Issue by ReMiii40** [dremin/RetroBar#434]
Hey,
I have a HiDPI screen, I have a scale of 175% and everything is blurry.
![retrobar](https://user-images.githubusercontent.com/40918334/188413836-4791720b-61e8-4a96-9d8c-91d4428a3a17.png)
I was able to make a custom theme to have the orb look good, but I have 0 idea for taskbar icons
And there is a transparent line on the bottom of the taskbar (only on my 4k screen)
![Sans titre 4](https://user-images.githubusercontent.com/40918334/188416281-ce3ca720-f6cb-422f-a4f4-117e88b880a6.png)
Regards

**Comment by xoascf** [comment]
To use large icons in your theme, you can use the following:
`<system:Boolean x:Key="UseLargeIcons">True</system:Boolean>`

**Comment by dremin** [comment]
Hmm, if I use 175% my icons are not nearly that fuzzy:
![Screenshot 2022-09-07 221712](https://user-images.githubusercontent.com/7373633/189026401-435f016b-2ab9-4a71-a394-49d5a3117eb6.png)

Do you have multiple monitors with different scales? If so what is the configuration of each?

**Comment by ReMiii40** [comment]
> 

Yep, 2 monitors (one 175% and one 100%), if I use only my external monitor (Win + P > "Second screen only") I've got a perfect taskbar (except the transparent line)
![Sans titre 2](https://user-images.githubusercontent.com/40918334/189063279-b595e67d-e91b-447a-99ed-cb68494c03fe.png)

**Comment by ReMiii40** [comment]
> To use large icons in your theme, you can use the following: `<system:Boolean x:Key="UseLargeIcons">True</system:Boolean>`

Perfect with 2 screens/scales
